isotovideo

```diff
@@ -58,7 +58,7 @@ use needle;
 use autotest ();
 use commands;
 use distribution;
-use testapi 'diag';
+use bmwqemu qw(diag fctwarn);
```

Comment:
that should be moved up to line 55 where the module is already used

Reply:
yep, agreed. Done